<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>278</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Username: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="userName">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Password: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="password">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="signinButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>SignIn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="signupButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>SignUp</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLOGIN">
    <property name="title">
     <string>LOGIN</string>
    </property>
   </widget>
   <addaction name="menuLOGIN"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
